poly-synth/index.html

```diff
@@ -55,7 +56,11 @@
     </ul>
     <p>
     The combined output of the voices can then be modulated by bitcrushing,
-    reverb, and delay effects.
+    reverb, and delay effects. If noise gate side chaining is active, then
+    the output of the synthesizer will ramp towards silence when the level of
+    the drum sample exceeds the specified threshold. The beat and its volume is
```

Comment:
Isn't this backwards?  If the drum sample exceeds the threshold, the synthesizer ramps up, not down to silence.